cartoframes/auth/credentials.py

```diff
@@ -210,18 +211,15 @@ def delete(cls, config_file=None):
         except OSError:
             warnings.warn('No credential file found at {}.'.format(path_to_remove))
 
-    def get_do_token(self):
-        """Returns the Data Observatory v2 token"""
+    def get_do_credentials(self):
+        """Returns the Data Observatory v2 credentials"""
 
         do_token_manager = DoTokenManager(self.get_api_key_auth_client())
-        token = do_token_manager.get()
-        if not token:
+        self._do_credentials = do_token_manager.get()  # don't cache it, every call here should refresh it
```

Comment:
There is no need to store this in `self._do_credentials`, isn't it?

Reply:
Right!